modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/OnboardingLoginOrSignUpPage.kt

```diff
@@ -157,6 +202,39 @@ private fun LogInButton(onClick: () -> Unit) {
     )
 }
 
+private object Artwork {
```

Comment:
I really like how you grouped these functions inside an `Artwork` object to give us a nice namespace for these related functions.